Amplify/Categories/DataStore/Model/Internal/ModelRegistry.swift

```diff
@@ -35,23 +35,21 @@ public struct ModelRegistry {
     }
 
     public static func register(modelType: Model.Type) {
-        register(modelName: modelType.modelName,
-                 modelSchema: modelType.schema,
-                 modelType: modelType) { (jsonString, jsonDecoder) -> Model in
+        register(modelType: modelType,
+                 modelSchema: modelType.schema) { (jsonString, jsonDecoder) -> Model in
             let model = try modelType.from(json: jsonString, decoder: jsonDecoder)
             return model
         }
     }
 
-    public static func register(modelName: ModelName,
+    public static func register(modelType: Model.Type,
```

Comment:
I've lost track of what we've published--I assume that this change in signature to a `public` method is not a breaking change because we haven't yet released the DataStore changes that start to use `ModelSchema` right?